lib/import-export-visitor.js

```diff
@@ -468,6 +525,8 @@ module.exports = class ImportExportVisitor extends Visitor {
 
         path.replace(this._buildExportDefaultStatement(dd));
       }
+
+      this.madeChanges = true;
```

Comment:
I guess you need one to indicate strict mode should be used and one to overwrite the intent of the parser.